packages/react-storage/src/components/StorageBrowser/views/Controls/Table.tsx

```diff
@@ -316,6 +316,7 @@ export const LocationDetailViewTable = ({
   // @TODO: This should be it's own component instead of using `useCallback`
   const renderRowItem: RenderRowItem<LocationItemData> = React.useCallback(
     (row, index) => {
+      const { prefix = '' } = current ?? {};
```

Comment:
If we are only using `current` in the body of `renderRowItem` think we can just destructure out `prefix` closer to line 201 and add a comment there explaining the need for the defaulting `prefix` and `path` to empty strings

Reply:
I think this was more to avoid having to track another item in the dependencies list when already tracking current. But I can make this change for now - this should all go away with table integration anyway